recipes/grapheme/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "grapheme" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grapheme-{{ version }}.tar.gz
+  sha256: 44c2b9f21bbe77cfb05835fec230bd435954275267fea1858013b102f8603cca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - grapheme
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/alvinlindstam/grapheme
+  summary: Unicode grapheme helpers
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+
+extra:
+  recipe-maintainers:
+    - Brainor
+    - alvinlindstam

```

Comment:
@alvinlindstam do you want to maintain this recipe as well?

Reply:
@alvinlindstam is the author of this package. I guess I can add him as a maintainer when I acquire his permission later.